roles/ssh-hardening/templates/openssh.conf.j2

```diff
@@ -0,0 +1,111 @@
+# {{ansible_managed}}
+
+# This is the ssh client system-wide configuration file.
+# See ssh_config(5) for more information on any settings used. Comments will be added only to clarify why a configuration was chosen.
+#
+# Basic configuration
+# ===================
+
+# Address family should always be limited to the active network configuration.
+AddressFamily {{ 'any' if network_ipv6_enable else 'inet' }}
```

Comment:
https://gist.github.com/RiFi2k/41f3874f9747c9a5e611b105e373100a

Cleaned up and edited text on ssh-hardening/defaults/main.yml hopefully it saves a little time for you.